src/CardPreview/CardPreview.js

```diff
@@ -7,7 +7,13 @@ import createWithComponent from '../utils/createWithComponent';
 
 const CardPreview = styled(
   createWithComponent(Box, 'div', {
-    ratio: 9 / 16
+    defaultProps: {
+      ratio: 9 / 16
+    },
+    propTypes: {
+      image: PropTypes.string.isRequired,
+      ...ratio.propType
```

Comment:
is it `propType` or `propTypes`?

Reply:
I saw `ratio.propTypes = _extends({}, ratioPadding.propTypes);` But I don't see anything for `propType`. Just curious.

Reply:
Good catch, I believe you are correct. This should be `ratio.propTypes`